### ISSUE TITLE
not really an issue

### ISSUE THREAD

**Issue by brunob** [nagisa/Feeds#26]
Hi i have a few question about this app : 

Are you searching alpha/beta testers for it ?

Does the app work without needing a google reader account ?

I've tried to install this app on ubuntu 12.04 and installed all the required packages : 

python3 python3-gi python3-lxml sqlite3 python3-all python-sqlite python3-sqlalchemy

The overs required packages are installed, and i get this error when i try to install the app : 

```
Traceback (most recent call last):
  File "setup.py", line 5, in <module>
    from trifle.utils.const import VERSION
  File "/home/bb/Bureau/Feeds/trifle/utils/__init__.py", line 6, in <module>
    from trifle.utils.sqlite import sqlite
  File "/home/bb/Bureau/Feeds/trifle/utils/sqlite.py", line 70, in <module>
    sqlite = SQLite(_sqlite_path)
  File "/home/bb/Bureau/Feeds/trifle/utils/sqlite.py", line 15, in __init__
    super(SQLite, self).__init__(daemon=True)
TypeError: __init__() got an unexpected keyword argument 'daemon'
```

I've read here https://aur.archlinux.org/packages/trifle/?setlang=fr that intltool package seems to be required so i installed this one too, but this doesn't solve the error.

Have you any clue about this type of error ?

Thanks ++

**Comment by nagisa** [comment]
@brunob 

> Are you searching alpha/beta testers for it ?

Not at the moment. I still need to find what to replace Google Reader with.
_AMENDMENT:_ I do however use this application for my daily reading needs so it should work for anyone interested too. As long as Google Reader works.

> Does the app work without needing a google reader account ?

Sadly not, and it is tracked in issue #25

> super(SQLite, self).**init**(daemon=True)

Apparently daemon argument to `threading.Thread` was added in python 3.3. This is a bug, I fixed it in dabe501b90, but there might be more of such incompatibilities as I don't tend to test with python below 3.3.

**Comment by brunob** [comment]
Hi @simukis and thanks for your answer.

> Apparently daemon argument to threading.Thread was added in python 3.3. This is a bug, I fixed it in dabe501, but there might be more of such incompatibilities as I don't tend to test with python below 3.3.

Ok i understand, ubuntu 12.04 propose only python 3.2.3, si that's why i get this bug.

Anyway, good luck for the dev of this app :)

++
